README.md

```diff
@@ -0,0 +1,23 @@
+# Opdrachtbeschrijving
+
+In deze repository vind je een aantal opdrachten die je kunt maken om te oefenen met lists en tuples in Python.
+Deze repository bestaat uit meerdere opdrachten van oplopende moeilijkheidsgraad. 
+
+Om deze opdracht te maken kun je op Github, onder de groene knop 'Code', deze repository downloaden als zip bestand en openen in je IDE
+(PyCharm).
+
+## Output controleren
+Gebruik de print() functie om de output van je code te controleren.
+
+## Code runnen
+Je kan je code runnen door je Python bestand te selecteren in PyCharm en op de groene play knop te drukken.
+
+Of je kan de terminal openen en het volgende commando uitvoeren (vervang `<naam-van-file.py>` met de naam van je Python bestand)
+```bash
+ python3 naam-van-file.py
+```
+
+(Zorg dat je in de goede directory zit in je terminal.)
```

Comment:
En wat is dan de goede directory? Super flauw, maar misschien kun je beter zeggen: Zorg dat je de de terminal hebt geopend in de root-map van het project.